CONTRIBUTING.md

```diff
@@ -231,6 +231,24 @@ Writing separate tests might be much better than making a single test that check
 
 Also, we should use driver methods as much as possible instead of testing commands directly via `RunCommand`.
 
+#### Integration tests naming guidelines
+
+1. Test files should be named after the command they test.
+   For example, `find_test.go` for testing the find command.
+   Test files belonging to the same commands group are organized into folders named after this group,
+   such as `aggregation`, `operations`, `diagnostic`, etc.
+2. If the file contains compatibility tests, add the `_compat` suffix. For example, `find_compat_test.go`.
+3. Test names should include the name of the command being tested.
+   For instance, `TestDistinct` for testing the distinct command.
```

Comment:
Finally, for `RunCommand` do we have a preference? :thinking: 

Currently we have `TestFindAndModifyCommand`, `TestCommandsFindAndModify`, `TestFindAndModifyRunCommand`, or just `TestFindAndModify` because findAndModify can only be run as `RunCommand`.

Reply:
Yeah, let's document this one too!

Reply:
But I don't know which one to use. Up to you to suggest, I'm always in for shortest name :see_no_evil: